### DISCUSSION TITLE
liquid sonic cinematic room 5.1 and audiogridder

### DISCUSSION THREAD

**Discussion by ericmusicstrasbourg** [apohl79/audiogridder#374] [Q&A]
Hello,

is it possible to use multi-channel reverb. liquid sonics cinematic room is a multi-channel reverb. can we work in 5.1 with audio gridder

all the best

eric

**Comment by apohl79** [comment]
> is it possible to use multi-channel reverb. liquid sonics cinematic room is a multi-channel reverb. can we work in 5.1 with audio gridder

not yet, currently only 1 or 2 channels are supported
